### PR TITLE
Production features and Heroku deployment

### LINKED ISSUE #11: Single command to add SECRET_KEY (before deployment)
```
heroku config:set SECRET_KEY=$(python -c 'import random; import string; print("".join([random.SystemRandom().choice("{}{}{}".format(string.ascii_letters, string.digits, string.punctuation)) for i in range(50)]))')
```

### PR DESCRIPTION
**Pull request by fjsj**
Also, closes #11 #3 #2 #1

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,16 +25,14 @@ npm-debug.log
...
 # Webpack
 /assets/bundles/*
-webpack-stats.json
+/webpack-stats.json
 
 # Sass
 .sass-cache
 *.map
 
-# Css
```
Comment (fjsj) [inline .gitignore]: IMO makes no sense to ignore all CSS

File: README.md [hunk 2]
```diff
@@ -1,22 +1,55 @@
...
 ```
+- [ ] Above: don't forget the `--extension` and `--name` params!
 - [ ] `pip install -r requirements-to-freeze.txt`
 - [ ] `pip freeze > requirements.txt`
 - [ ] `npm update --save`
 - [ ] Check for outdated npm dependencies with `npm outdated` and update them
 - [ ] Change the first line of README to the name of the project
 - [ ] Create a Google Group to receive 500 errors and forward to your email. It should be a private group, but remember to change settings to allow posting from any email
 - [ ] Add the Google Group email address to the `ADMINS` settings variable
-- [ ] Create a [Sendgrid](https://sendgrid.com/) account and remember to configure your production environment so you can receive the error emails
```
Comment (fjsj) [inline README.md]: Added automatically as a Heroku plugin

File: app.json [hunk 3]
```diff
@@ -0,0 +1,67 @@
+{
+  "name": "{{project_name}}",
+  "description": "{{project_name|title}} Heroku app.",
+  "scripts": {
+    "postdeploy": "python manage.py migrate"
```
Comment (fjsj) [inline app.json]: This occurs only after the first deploy

File: circle.yml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+  override:
+    - npm install
+    - npm run build
+    - flake8
+    - >
+        DJANGO_SETTINGS_MODULE={{project_name}}.settings.local
+            python manage.py has_missing_migrations --ignore authtools;
+    - coverage run manage.py test
+  post:
+    - coverage html -d $CIRCLE_ARTIFACTS
```
Comment (fjsj) [inline circle.yml]: No continuous deployment yet

File: project_name/settings/base.py [hunk 5]
```diff
@@ -95,10 +100,15 @@ def base_dir_join(*args):
 # Webpack
 WEBPACK_LOADER = {
     'DEFAULT': {
-        'CACHE': True,
+        'CACHE': False,
```
Comment (fjsj) [inline project_name/settings/base.py]: CACHE should be actually False in local and True in production  
https://github.com/owais/django-webpack-loader#default-configuration

File: project_name/settings/base.py [hunk 6]
```diff
@@ -95,10 +100,15 @@ def base_dir_join(*args):
...
         'STATS_FILE': base_dir_join('webpack-stats.json'),
         'POLL_INTERVAL': 0.1,
         'IGNORE': ['.+\.hot-update.js', '.+\.map']
     }
 }
+
+# Celery
+CELERY_ACCEPT_CONTENT = ['json']
+CELERY_TASK_SERIALIZER = 'json'
+CELERY_RESULT_SERIALIZER = 'json'
```
Comment (fjsj) [inline project_name/settings/base.py]: Avoids security problems, pickle is deprecated. But reduces flexibility. All task params must be JSON serializable

File: project_name/settings/production.py [hunk 7]
```diff
@@ -14,30 +13,61 @@
...
 
 EMAIL_HOST = 'smtp.sendgrid.net'
-EMAIL_HOST_USER = config('EMAIL_HOST_USER')
-EMAIL_HOST_PASSWORD = config('EMAIL_HOST_PASSWORD')
+EMAIL_HOST_USER = config('SENDGRID_USERNAME')
+EMAIL_HOST_PASSWORD = config('SENDGRID_PASSWORD')
 EMAIL_PORT = 587
 EMAIL_USE_TLS = True
 
+# Security
```
Comment (fjsj) [inline project_name/settings/production.py]: Makes prod HTTPS only

**Comment by aericson** [comment]
LGTM

**Comment by filipeximenes** [comment]
LGTM

**Pull request merged**